src/Symfony/Component/Workflow/Attribute/AsAnnounceListener.php

```diff
@@ -14,13 +14,22 @@
 use Symfony\Component\EventDispatcher\Attribute\AsEventListener;
 
 /**
+ * An attribute to define a listener for the "announce" event of a workflow.
```

Comment:
not need to state the obvious:
```suggestion
 * Defines a listener for the "announce" event of a workflow.
```

Reply:
Updated and added to missing ones